ops/packer/scripts/build_bfd-docker-host.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I believe this is still the preferred, _more_ portable shebang. However, I've recently read that this isn't so cut and dry, and some really **do** prefer the simpler variant that you've offered here for fear of `env` not being available. I'll leave it up to you as to whether you want to follow this or an alternative ¯\\\_(ツ)_/¯ 

```suggestion
#!/usr/bin/env bash
```

Reply:
I'm tempted to leave this as-is and change it in 2887 so I don't need to re-test the AMI creation (which is probably unnecessary but still...better safe than sorry)